molo/core/content_migration/helper.py

```diff
@@ -0,0 +1,164 @@
+import json
+
+from django.shortcuts import get_object_or_404
+
+from molo.core.models import (
+    Main, SiteLanguage, PageTranslation, SectionPage, ArticlePage, FooterPage)
+
+from unicore.content.models import Category, Page
+
+
+class ContentImportHelper(object):
+
+    def __init__(self, ws):
+        self.ws = ws
+
+    def get_or_create(self, cls, obj, parent):
+        if cls.objects.filter(uuid=obj.uuid).exists():
+            instance = cls.objects.get(uuid=obj.uuid)
+            instance.title = obj.title
+            print 'updated', obj.title
+            return instance
+
+        instance = cls(uuid=obj.uuid, title=obj.title)
+        parent.add_child(instance=instance)
+        print 'created', obj.title
+        return instance
+
+    def get_or_create_translation(self, cls, obj, parent, language):
+        if cls.objects.filter(uuid=obj.uuid).exists():
+            instance = cls.objects.get(uuid=obj.uuid)
+            instance.title = obj.title
+            language_relation = instance.languages.first()
+            language_relation.language = language
+            language_relation.save()
+            print 'updated translation', obj.title
+            return instance
+
+        instance = cls(uuid=obj.uuid, title=obj.title)
+        parent.add_child(instance=instance)
+        language_relation = instance.languages.first()
+        language_relation.language = language
+        language_relation.save()
+        print 'created translation', obj.title
+        return instance
+
+    def import_content_for(self, locales):
+        for selected_locale in locales:
+            site_language = get_object_or_404(
+                SiteLanguage, locale=selected_locale.get('site_language'))
+
+            if site_language.is_main_language:
```

Comment:
@saeedmb8 so everything in this IF block can move to a new method called `def import_main_language_content(self, site_language):`